helm_deploy/values-prod.yaml

```diff
@@ -16,6 +16,7 @@ generic-service:
     HMPPS_PRISON_API_URL: "https://api.prison.service.justice.gov.uk"
     NOMIS_USER_ROLES_API_URL: "https://nomis-user.aks-live-1.studio-hosting.service.justice.gov.uk"
     TOKEN_VERIFICATION_API_URL: "https://token-verification-api.prison.service.justice.gov.uk"
+    FEATURE_HIDE_DAYS_COLUMNS_IN_INCENTIVES_TABLE: "false"
```

Comment:
why false in prod? Is to validate first?

Reply:
Igor added some utility to read feature flags values and [it would default to false anyway](https://github.com/ministryofjustice/hmpps-incentives-ui/compare/ag--INC-385-hide-days-columns?expand=1#diff-b5e32f68bcc433506f573cc64aae69018f0a1caa3822686806a704ac1c8e906cR122) but I thought it would be good to be explicit and spell it out even if the effect is the same.